tests/network-ovn

```diff
@@ -216,11 +216,12 @@ ovn_basic_tests() {
     lxc network unset dummy ipv6.routes.anycast --project default
 
     lxc init "${IMAGE}" u1 --project testovn -s default
-    lxc config device add u1 eth0 nic network=ovn-virtual-network name=eth0 --project testovn
 
-    echo "==> Record NAT rules count before u1 started"
+    echo "==> Record NAT rules count before u1 eth0 NIC added"
     natRulesBefore=$(ovn-nbctl --bare --format=csv --column=external_ip,logical_ip,type find nat | wc -l)
+    ovn-nbctl --bare --format=csv --column=external_ip,logical_ip,type find nat
```

Comment:
```suggestion
    out="$(ovn-nbctl --bare --format=csv --column=external_ip,logical_ip,type find nat)"
    natRulesBefore="$(wc -l <<< "${out}")"
```
The above pattern would avoid dup'ing the commands (and get them de-sync'ed) and since we use `set -x`, we'd still get the output shown in the logs:

```
$ set -x
$ out=$(cat /etc/hosts)                                                                                   
++ cat /etc/hosts
+ out='127.0.0.1	localhost
127.0.1.1	sdeziel-lemur.sdeziel.info sdeziel-lemur

# The following lines are desirable for IPv6 capable hosts
::1     ip6-localhost ip6-loopback
fe00::0 ip6-localnet
ff00::0 ip6-mcastprefix
ff02::1 ip6-allnodes
ff02::2 ip6-allrouters'
```

Reply:
@simondeziel have fixed this now.